### ISSUE TITLE
Build pipeline for linux?

### ISSUE THREAD

**Issue by hartmark** [jwt-dotnet/jwt#407]
I see that there are a pipeline setup for the repo, it does however just build on windows.

Regarding the move to System.Text.Json and the question of increasing test coverage I think it would be nice to have added coverage to also build for linux.

Thoughts?

**Comment by abatishchev** [comment]
Would be awesome! To at least run tests in parallel on two different pools/OS.

**Comment by hartmark** [comment]
I'm running on arch linux at home and windows at work so I can assist if there is some issues :)

**Comment by hartmark** [comment]
Where do I find the builds?

**Comment by abatishchev** [comment]
https://dev.azure.com/abatishchev/OpenSource/_build?definitionId=9&_a=summary

**Comment by hartmark** [comment]
I see only Windows, for example
https://dev.azure.com/abatishchev/OpenSource/_build/results?buildId=1166&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=c8d7d21c-d114-4864-a7cc-744b61100477

**Comment by abatishchev** [comment]
I remember a pipeline was using Ubuntu but actually it's the deployment that uses it, not built. Sorry, close too fast.

**Comment by lahma** [comment]
Would a solution utilizing NUKE build and running on native GitHub Actions be accepted? The build produce the same artifacts as before.

**Comment by abatishchev** [comment]
I'd like to continue using ADO, I like it much more.

I think all the current pipeline missing is a matrix which would make run the same build on two different platform in parallel.
